protocol/communities/manager.go

```diff
@@ -18,23 +21,37 @@ import (
 
 	"github.com/status-im/status-go/eth-node/crypto"
 	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/params"
 	"github.com/status-im/status-go/protocol/common"
 	"github.com/status-im/status-go/protocol/ens"
 	"github.com/status-im/status-go/protocol/protobuf"
 	"github.com/status-im/status-go/protocol/requests"
+	"github.com/status-im/status-go/protocol/transport"
 )
 
+var defaultAnnounceList = [][]string{
+	{"udp://tracker.opentrackr.org:1337/announce"},
+	{"udp://tracker.openbittorrent.com:6969/announce"},
+}
+var pieceLength = 100 * 1024
```

Comment:
Going with 100kb pieces now but we probably want to change this @staheri14 I believe you wanted to propose a good piece size?